src/components/home/elements/DepositToken.vue

```diff
@@ -0,0 +1,634 @@
+                }
+                apiSchema = dataInfo
+            } else if (walletType === 'coldWallet') {
+                const coldDataInfo = {
+                    contractId: this.contractId,
+                    senderPublicKey: this.coldAddresses[this.address].publicKey,
+                    fee: CONTRACT_EXEC_FEE * VSYS_PRECISION,
+                    feeScale: FEE_SCALE,
+                    timestamp: this.dataObject.timestamp,
+                    functionIndex: this.isSplit ? DEPOSIT_FUNCIDX_SPLIT : DEPOSIT_FUNCIDX,
+                    functionData: this.dataObject.function,
+                    signature: this.coldSignature
+                }
+                apiSchema = coldDataInfo
+            }
+            const url = NODE_IP + '/contract/broadcast/execute'
+            this.$http.post(url, apiSchema).then(response => {
+                if (walletType === 'hotWallet') {
+                    this.pageId++
+                } else {
+                    this.coldPageId++
+                }
+            }, response => {
+                this.errorMessage = response.body.message
+                if (this.errorMessage === undefined) {
+                    this.errorMessage = 'Failed reason: Unknown.Please check network connection!'
+                }
+                this.sendError = true
+            })
+        },
+        nextPage: function() {
+            this.timeStamp = Date.now() * 1e6
+            this.hasConfirmed = false
+            this.pageId++
+        },
+        coldNextPage: function() {
+            this.sendError = false
+            this.coldPageId++
+        },
+        prevPage: function() {
+            this.sendError = false
+            if (this.pageId === 1) {
+                this.$refs.sendModal.hide()
+            } else {
+                this.pageId--
+            }
+        },
+        coldPrevPage: function() {
+            this.sendError = false
+            if (this.coldPageId === 1) {
+                this.$refs.sendModal.hide()
+            } else {
+                this.coldPageId--
+            }
+        },
+        resetPage: function() {
+            this.amount = BigNumber(0)
+            this.pageId = 1
+            this.coldPageId = 1
+            this.paused = false
+            this.sendError = false
+            this.coldSignature = ''
+            this.contractId = ''
+        },
+        endSend: function() {
+            for (let delayTime = 6000; delayTime < 30100; delayTime *= 5) { //  Refresh interval will be 6s, 30s, 150s
+                setTimeout(this.sendBalanceChange, delayTime)
```

Comment:
Use setInterval. Save and operate the interval

Reply:
I think use `setTimeout` is better here if refresh in 6s, 30s, 150s. Because it just run one time. `setInterval` will keep running in loop.